storage/replica_test.go

```diff
@@ -1588,6 +1589,69 @@ func TestRangeResponseCacheStoredError(t *testing.T) {
 	}
 }
 
+// TestEndTransactionDeadline verifies that EndTransaction respects the
+// transaction deadline.
+func TestEndTransactionDeadline(t *testing.T) {
```

Comment:
isn't the corresponding test in `kv` just a one-to-one copy of this one? Is there a reason for keeping both?